GitUI/Translation/English.xlf

```diff
@@ -1557,7 +1557,7 @@ You will have push access. {2}</source>
         <target />
       </trans-unit>
       <trans-unit id="label3.Text">
-        <source>Add remote as:</source>
```

Comment:
Is this really the only translation related change?
I expected that the renamed controls and at least "Forks" -> "# Forks" to show here.

Reply:
Thanks for this remarks that make me see that some columns were not translated.
I fixed it.